Ref/SignalGen/SignalGen.fpp

```diff
@@ -92,6 +86,8 @@ module Ref {
     # ----------------------------------------------------------------------
 
     include "Events.fppi"
+    include "../Interfaces/EventsInterface.fppi"
```

Comment:
This is the events section where events are to be defined. Interfaces should likely be their own section, or another type of grouping.
@LeStarch do we have a convention for this? Should we?

Reply:
Also we probably want to do the same change on all components across `Ref/` and `RPI/`

Reply:
I agree. We should create an interfaces section and place this there.

Also, `"../Interfaces/EventsInterface.fppi"` should read `"../../Fw/Interfaces/EventsInterface.fppi"`